src/udev/udev-varlink.c

```diff
@@ -3,6 +3,17 @@
 #include "udev-manager.h"
 #include "udev-varlink.h"
 
+static int vl_method_ping(Varlink *link, JsonVariant *parameters, VarlinkMethodFlags flags, void *userdata) {
+        assert(link);
+
+        if (json_variant_elements(parameters) > 0)
+                return varlink_error_invalid_parameter(link, parameters);
+
+        log_debug("Received io.systemd.udev.Ping");
+
+        return varlink_reply(link, NULL);
+}
```

Comment:
I'd implement this in generic code in varlink.c, for the aforementioned reason that the concept is generic enough to just be implemented everyhwere.